<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWin</class>
 <widget class="QWidget" name="CMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>488</width>
    <height>192</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CMainWin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_iSrcInfoEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_iSrcInfoLbl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>原始字符串：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="m_iEncryptRBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>加密</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_iDecryptRBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>解密</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>密钥：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>密钥类型：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_iKeyEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IV：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="m_iKeyLenghtEdit">
        <item>
         <property name="text">
          <string>AES128</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES192</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES256</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_iIvecEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label3_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>padding类型：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="m_iPaddingTypeEdit">
        <item>
         <property name="text">
          <string>PKCS5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PKCS7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOPADDING</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_iDstInfoLbl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>加密字符串：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="m_iDstInfoEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="m_iStartBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>加密</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_iKeyEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_Key_Changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iIvecEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_Ivec_Changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iEncryptRBtn</sender>
   <signal>clicked()</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_Encrypt_Decrypt_Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iDecryptRBtn</sender>
   <signal>clicked()</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_Encrypt_Decrypt_Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iKeyEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_Key_Enter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iSrcInfoEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_SrcString_Changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iStartBtn</sender>
   <signal>clicked()</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_StartBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iSrcInfoEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>CMainWin</receiver>
   <slot>__On_SrcString_Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>__On_Key_Changed(QString)</slot>
  <slot>__On_Ivec_Changed(QString)</slot>
  <slot>__On_Encrypt_Decrypt_Changed()</slot>
  <slot>__On_Key_Enter()</slot>
  <slot>__On_SrcString_Changed(QString)</slot>
  <slot>__On_StartBtn_Clicked()</slot>
  <slot>__On_SrcString_Changed()</slot>
 </slots>
</ui>
